test/e2e/storage/volume_limits.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package storage
+
+import (
+	. "github.com/onsi/ginkgo"
+	"k8s.io/api/core/v1"
+	clientset "k8s.io/client-go/kubernetes"
+	v1helper "k8s.io/kubernetes/pkg/apis/core/v1/helper"
+	"k8s.io/kubernetes/test/e2e/framework"
+	"k8s.io/kubernetes/test/e2e/storage/utils"
+)
+
+var _ = utils.SIGDescribe("Volume limits", func() {
+	var (
+		c clientset.Interface
+	)
+	f := framework.NewDefaultFramework("volume-limits-on-node")
+	BeforeEach(func() {
+		framework.SkipUnlessProviderIs("aws", "gce")
+		c = f.ClientSet
+		framework.ExpectNoError(framework.WaitForAllNodesSchedulable(c, framework.TestContext.NodeSchedulableTimeout))
+	})
+
+	It("should verify that all nodes have volume limits", func() {
+		nodeList := framework.GetReadySchedulableNodesOrDie(f.ClientSet)
+		if len(nodeList.Items) == 0 {
+			framework.Failf("Unable to find ready and schedulable Node")
+		}
+		node := nodeList.Items[0]
+		volumeLimits := getVolumeLimit(&node)
+		if len(volumeLimits) == 0 {
+			framework.Failf("Expected volume limits to be set")
+		}
```

Comment:
can we also launch a pod that uses the entire volume limit?  You could potentially reuse this test case and just set a different numVolumes: https://sourcegraph.com/github.com/kubernetes/kubernetes/-/blob/test/e2e/storage/persistent_volumes.go#L319

Reply:
We could but for gce - it appears that the limit on the selected node starts at 64. So, we will have to create a pod with 64+ volumes

Reply:
64 1gi volumes should be within our quota...

Reply:
okay, I think that will also make this test serial I think because no other workload(guess just with volumes but we don't have that level of granularity) can be scheduled on the node while this test runs on it.  I can do that as a follow up PR while fixing https://github.com/kubernetes/kubernetes/issues/68912